src/modules/drive/FabWithAddMenuContext.jsx

```diff
@@ -20,7 +20,7 @@ const FabWithAddMenuContext = ({ noSidebar }) => {
   } = useContext(AddMenuContext)
 
   const styles = useFabStyles({
-    bottom: noSidebar ? '1rem' : 'calc(var(--sidebarHeight) + 2rem)'
+    bottom: noSidebar ? '4rem' : 'calc(var(--sidebarHeight) + 2rem)'
```

Comment:
is it the right height on Cozy and Twake theme?

Reply:
Nevermind, we don't maintain Cozy theme anymore, you just can make changes for Twake theme purposes 👍